bucket/foxit-reader.json

```diff
@@ -6,26 +6,27 @@
         "identifier": "Freeware",
         "url": "https://www.foxitsoftware.com/pdf-reader/eula.html"
     },
-    "url": "https://cdn01.foxitsoftware.com/pub/foxit/reader/desktop/win/9.x/9.7/en_us/FoxitReader97_Setup_Prom_IS.exe#/dl.7z",
-    "hash": "341e29058ee6ba4e952980fe07e68a36579706f5116fda0e26736f11809a96d7",
+    "url": "https://cdn01.foxitsoftware.com/product/reader/desktop/win/9.7/D205F785AF753E06041933E5C5E19E5C/FoxitReader97_enu_Setup_Prom.exe#/dl.7z",
+    "hash": "md5:d205f785af753e06041933e5c5e19e5c",
     "bin": "FoxitReader.exe",
     "shortcuts": [
         [
             "FoxitReader.exe",
             "Foxit Reader"
         ]
     ],
-    "installer": {
-        "script": [
-            "Get-ChildItem \"$dir\\`$PLUGINSDIR\" '*_Prom.exe' | Select-Object -ExpandProperty Fullname | Expand-InnoArchive -DestinationPath \"$dir\"",
-            "Remove-Item \"$dir\\`$PLUGINSDIR\" -Recurse"
-        ]
+    "innosetup": {
+        "type": true
     },
     "checkver": {
         "url": "https://www.foxitsoftware.com/downloads/downloadForm.php?retJson=1&product=Foxit-Reader&platform=Windows&language=English",
-        "jsonpath": "$.package_info.version[0]"
```

Comment:
It's still returned as JSON, why have you switched to regex?

Reply:
The full path to the file contains the md5 hash. I have not found a way to get and version and hash through JSON.

Reply:
Ok.